src/tutorial/context.rst

```diff
@@ -0,0 +1,56 @@
+.. _tutor-context:
+
+Context creation
+================
+
+A context is an object that allows palace to access OpenAL,
+which is essential when you work with palace. Context maintains
+the audio environment and contains environment settings and components
+such as sources, buffers, and effects.
+
+Creating a device object
+------------------------
+
+To create a context, we must first create a device,
+since it's a parameter of the context object.
+
+To create an object, well, you just have to instantiate
+the `Device` class.
+
+.. code-block:: python
+
+   from palace import Device
+
+   with Device() as dev:
+       # Your code goes here
+
+This is how you declare a `Device` object with the default device.
+There can be several devices available, which can be found
+in `palace.device_names`.
+
+Creating a context
+------------------
+
+Now that we've created a device, we can create the context:
+
+.. code-block:: python
+
+   from palace import Device, Context
+
+   with Device() as dev, Context(dev) as ctx:
+       # Your code goes here
+
+Before we continue, if we only use `dev` for context creation,
+is there a shorter way to write it? The answer is yes.
+
+.. code-block::
+
+   from palace import Device, Context
+
+   with Device(), Context() as ctx:
```

Comment:
Missing highlight, also `Context()` does not work (yet, and I don't think it's a good idea either).